corehq/apps/app_manager/xpath_validator/wrapper.py

```diff
@@ -16,8 +16,8 @@ def validate_xpath(xpath, allow_case_hashtags=False):
         else:
             cmd = ['node', path]
         p = subprocess.Popen(cmd, stdin=subprocess.PIPE, stdout=subprocess.PIPE,
-                             stderr=subprocess.PIPE)
-        stdout, stderr = p.communicate(xpath.replace('\r', ' ').encode('utf-8'))
+                             stderr=subprocess.PIPE, universal_newlines=True)
```

Comment:
I think you also want `encoding='utf-8'` here (in addition to `universal_newlines=True`). The [documentation says](https://docs.python.org/3.6/library/subprocess.html#frequently-used-arguments):

> When `universal_newlines` is `True`, the class uses the encoding `locale.getpreferredencoding(False)` instead of `locale.getpreferredencoding()`. See the `io.TextIOWrapper` class for more information on this change.